steadybit/README.md

```diff
@@ -0,0 +1,67 @@
+# Steadybit
+
+## Overview
+
+[Steadybit][1] is a chaos engineering platform allowing you to simulate turbulent conditions in a controlled way, to improve system reliability and guide your organization to better incident management.
+
+The Steadybit integration uses the status of Datadog monitors within chaos engineering experiments. This integration provides your team insight into chaos engineering activity such as the experiments environment, start and end time, and the result of the experiment through Datadog events.
+
+## Setup
+
+The integration between Datadog and Steadybit is done through the [Steadybit Datadog extension][2]. The extension interacts with Datadog's API to gather information about monitors and report events to Datadog.
+
+### Prerequisites
+
+You need a [free or paid Steadybit license][3]. The integration supports Steadybit's SAAS and on-premises offering.
+
+### Installation
+
+Several [installation methods are supported][4]. For the best experience, install the Steadybit Datadog extension through the dedicated Helm chart.
+
+To learn more about the supported values for `datadog.siteParameter` and `datadog.siteUrl`, see the [Datadog sites][5] page.
+
+```
```

Comment:
Can you please add a line here explaining what the following text represents/what the user should do with it? Are these commands that the user needs to run to install?

Reply:
Made some slight wording changes and merged the two paragraphs (since the paragraph above already explains this).